TailLight/debug.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
+
+#pragma once
+
+#define TRACE_FN_ENTRY KdPrintEx((DPFLTR_IHVDRIVER_ID, DPFLTR_TRACE_LEVEL, "TailLight: Entry %s\n", __func__));
```

Comment:
Please don't bundle changes to driver tracing/logging with the other changes in this PR. Feel free to instead submit it in a _separate_ PR if you want changes in this area.

Reply:
I put a #pragma message and a TODO in to remove this before checkin.

Reply:
I don't see any changes. The `TRACE_FN_ENTRY` define is stil present in this pull request.

Reply:
Thanks.